### ISSUE TITLE
Show tooltip when hovering over result icon

### ISSUE THREAD

**Issue by CharliePoole** [TestCentric/testcentric-gui#1510]
For better accessibility, we should show a tooltip when hovering over a result icon, indicating whether it indicates Passed, Failed, etc. This is not needed in cases where the icon is already accompanied by a label, but only where it stands on its own.

These icons occur in various places in the GUI...

- In the tree itself (tooltip needed)
- In the test results tab for suites (already labelled, so no tooltip needed)
- In the status bar (tooltip needed)
- In the test Filter Display (tooltips already present)
- On the Tree Display page of the Settings Dialog (tooltips or labels needed)
- Other?

In the case of the tree, it may be necessary or desirable to apply the tooltip to the node itself. Some experimentation will be needed here.

**Comment by CharliePoole** [comment]
Reopened while I continue to improve the solution

**Comment by CharliePoole** [comment]
Fixed in PR #1513
